paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/PlaygroundSettings.kt

```diff
@@ -199,6 +200,12 @@ internal class PlaygroundSettings private constructor(
             playgroundState: PlaygroundState.Payment,
             configurationData: PlaygroundSettingDefinition.EmbeddedConfigurationData,
         ) {
+            val embeddedAppearance =
```

Comment:
👎 I think this is the "cleanest" hack for the meantime. I could also add this to AppearanceBottomSheetDialogFragment.onDestroy, or in the fragment result listener on PaymentSheetPlaygroundActivity

Reply:
Can this be added to value updated or similar in the definition instead?

Reply:
Updated 👍